dashboard/test/factories/pd_factories.rb

```diff
@@ -531,4 +531,55 @@
     association :regional_partner
     state 'WA'
   end
+
+  factory :pd_facilitator_application1819_hash, class: 'Hash' do
+    initialize_with do
+      {
+        firstName: 'Rubeus',
+        lastName: 'Hagrid',
+        phone: '555-555-5555',
+        address: '101 Hogwarts Ave',
+        city: 'Seattle',
+        state: 'WA',
+        zipCode: '98101',
+        genderIdentity: 'Male',
+        race: ['White'],
```

Comment:
I think this would technically be OTHER (half-giant)... :)

Reply:
Indeed. Will update :)

Reply:
I believe that would actually be species, which we do not ask currently.

Reply:
Since humans and giants can interbreed, they are the same species.